src/ATen/native/xpu/XPUFallback.template

```diff
@@ -171,6 +171,7 @@ TORCH_LIBRARY_IMPL(aten, XPU, m) {
     "frexp.Tensor_out",
     "_fused_moving_avg_obs_fq_helper",
     "geqrf",
+    "heaviside.out",
```

Comment:
Should we remove `i0.out` from fallback list?

Reply:
Thanks for pointing out. I guess this is due to some merge conflicts not being properly resolved. Will remove i0.